Fix 1441 minimum needs result

ismailsunni opened the pull request:
For #1441

Charlotte-Morgan commented:
hi @ismailsunni - this is looking good.
- [ ] I have looked at flood raster on population raster with aggegation (jakarta) and its fine. 
so in this case affected = evacuated = basis for minimum needs. I'd like to see rounding in the general & detailed reports. But that can wait for IF review. we probably need something that states that all affected people are assumed to be evacuated etc
- [ ] i have looked at flood polygon on population raster with aggregation (jakarta). In this case evacuated = 1% of affected and so the numbers in the reports are different. I like the extra information you include in the detailed minimum needs report (for people needing evacuation). Things get messy though with the detailed gender report and the detailed age report. Both are for affected and not the evacuated population; but then hygiene packs and additional rice are calculated on the affected rather than evacuated population. It would be helpful to add (affected people) to the title for the detailed gender and detailed age reports. Please add this to things to be included in the IF review
# cc
@adissadis

ismailsunni commented:
For point 1, basically we want to do it in IF review.
I will put in the notes that all affected people are assumed to be evacuated for now. So, for flood raster on population raster, it will give report like:
![selection_037](https://cloud.githubusercontent.com/assets/1421861/5952538/b9b7823c-a7ab-11e4-9695-b21cb4f477a8.png)


For point 2:
For flood polygon and population raster.
I have added extra notes in the table header. See below:
![selection_038](https://cloud.githubusercontent.com/assets/1421861/5952548/fefa0c5c-a7ab-11e4-8fb8-7395413e3aa2.png)

Is that ok?

Actually, I added the same table header for population-IF, for the detailed table. To make them uniform.

timlinux commented on safe/impact_statistics/aggregator.py:
```diff
@@ -949,27 +946,27 @@ def _aggregate_line_impact(self, safe_impact_layer):
-                # 2, # Ellipsoidal
-                0,  # Layer CRS
-                tmp_filename)
+                                         projected_layer,
```
it was correctly indented above (pull left!)

timlinux commented:
Great that is nice and clear thanks! LGTM
